From: Georges Khaznadar <georgesk@debian.org>
Date: Sat, 3 Sep 2016 17:57:10 +0200
Subject: Adjusting content in the desktop-files

Almost only extending / modifying the 'Keyword' values to catch up more
topics.
---
 resources/linux/launchers/org.kicad.bitmap2component.desktop.in | 4 ++--
 resources/linux/launchers/org.kicad.gerbview.desktop.in         | 2 +-
 resources/linux/launchers/org.kicad.kicad.desktop.in            | 4 ++--
 resources/linux/launchers/org.kicad.pcbcalculator.desktop.in    | 2 +-
 resources/linux/launchers/org.kicad.pcbnew.desktop.in           | 4 ++--
 5 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/resources/linux/launchers/org.kicad.bitmap2component.desktop.in b/resources/linux/launchers/org.kicad.bitmap2component.desktop.in
index 5e8cc6b..0e11b64 100644
--- a/resources/linux/launchers/org.kicad.bitmap2component.desktop.in
+++ b/resources/linux/launchers/org.kicad.bitmap2component.desktop.in
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-Icon=@KICAD_DESKTOP_FILE_ICON_PREFIX@bitmap2component
-Keywords=bitmap2component;Bitmap;Converter;
+Icon=bitmap2component
+Keywords=Bitmap;Converter;
 Type=Application
 Categories=Science;Electronics;
 Exec=bitmap2component %f
diff --git a/resources/linux/launchers/org.kicad.gerbview.desktop.in b/resources/linux/launchers/org.kicad.gerbview.desktop.in
index d10f4ee..98ce38d 100644
--- a/resources/linux/launchers/org.kicad.gerbview.desktop.in
+++ b/resources/linux/launchers/org.kicad.gerbview.desktop.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-Icon=@KICAD_DESKTOP_FILE_ICON_PREFIX@gerbview
+Icon=gerbview
 Type=Application
 Categories=Science;Electronics;
 Exec=gerbview %F
diff --git a/resources/linux/launchers/org.kicad.kicad.desktop.in b/resources/linux/launchers/org.kicad.kicad.desktop.in
index 57ba467..2417523 100644
--- a/resources/linux/launchers/org.kicad.kicad.desktop.in
+++ b/resources/linux/launchers/org.kicad.kicad.desktop.in
@@ -1,12 +1,12 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-Icon=@KICAD_DESKTOP_FILE_ICON_KICAD@
+Icon=kicad
 Type=Application
 Categories=Science;Electronics;
 Exec=kicad %f
 StartupWMClass=kicad
-Keywords=Projectmanager;
+Keywords=KiCad;Projectmanager;
 MimeType=application/x-kicad-project;
 Name=KiCad
 GenericName=EDA Suite
diff --git a/resources/linux/launchers/org.kicad.pcbcalculator.desktop.in b/resources/linux/launchers/org.kicad.pcbcalculator.desktop.in
index a5d7523..54a060d 100644
--- a/resources/linux/launchers/org.kicad.pcbcalculator.desktop.in
+++ b/resources/linux/launchers/org.kicad.pcbcalculator.desktop.in
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-Icon=@KICAD_DESKTOP_FILE_ICON_PREFIX@pcbcalculator
+Icon=pcbcalculator
 Type=Application
 Categories=Science;Electronics;
 Exec=pcb_calculator
diff --git a/resources/linux/launchers/org.kicad.pcbnew.desktop.in b/resources/linux/launchers/org.kicad.pcbnew.desktop.in
index 470360c..a46a61a 100644
--- a/resources/linux/launchers/org.kicad.pcbnew.desktop.in
+++ b/resources/linux/launchers/org.kicad.pcbnew.desktop.in
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-Keywords=pcbnew;PCB;Circuit;Layout;Design;Editor;Footprint;
-Icon=@KICAD_DESKTOP_FILE_ICON_PREFIX@pcbnew
+Keywords=PCB;Circuit;Layout;Design;Editor;Footprint;
+Icon=pcbnew
 Type=Application
 Categories=Science;Electronics;
 Exec=pcbnew %f
